java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -256,6 +261,10 @@ class DatabaseImpl(
         val singletons = mutableMapOf<FieldName, Referencable?>()
         val collections = mutableMapOf<FieldName, MutableSet<Referencable>>()
 
+        val numeric_types = "(${PrimitiveType.Number.ordinal}, " +
```

Comment:
you need to update clearEntities() to also use this here: https://source.corp.google.com/piper///depot/google3/third_party/java_src/arcs/java/arcs/android/storage/database/DatabaseImpl.kt;rcl=312400010;l=864

Reply:
Done, thanks! I added some extra bits to one of the tests too.